.buildkite/pipeline.yml

```diff
@@ -65,3 +70,17 @@ steps:
 
     - label: "contracts/eth/"
       command: buildkite-agent pipeline upload .buildkite/contracts-eth.yml
+
+    - label: publish docker
+      command: ./utils/scripts/docker-release.sh
+      plugins:
+      - docker-login#v2.0.1:
+          username: nearprotocol
+          password-env: DOCKER_LOGIN_PASSWORD
+      branches: "master | mhala-docker"
```

Comment:
should this be only `master`?

Reply:
Yes, left it here only to test out this PR